.github/workflows/sigbuild-docker-branch.yml

```diff
@@ -79,3 +79,34 @@ jobs:
         name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
 
+  provenance-dockerhub:
+    needs: [docker]
+    if: github.repository == 'tensorflow/tensorflow' # Don't do this in forks
+    permissions:
+      actions: read # for detecting the Github Actions environment.
+      id-token: write # for creating OIDC tokens for signing.
+      packages: write # for uploading attestations.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@main  # TODO(mihaimaruseac): Pin and use signed releases when they exist
+    with:
+      image: ${{ needs.build.outputs.tag }}
```

Comment:
Is this `build` rather than the step id, `docker_build`?

Reply:
Yep, fixed